Assets/Project/Scripts/Common/Networks/Requests/GetStageStatsRequest.cs

```diff
@@ -23,7 +23,10 @@ protected override object DeserializeResponse()
 
         protected override object GetData_Local()
         {
-            throw new System.NotImplementedException();
+            // デバッグ用
+            var stageStats = new StageStats(clearRate: 20f, minClearTime: 1f, stageID: "1-1");
```

Comment:
```suggestion
            var stageStats = new StageStats(clearRate: 0.2f, minClearTime: 1f, stageID: "1-1");
```
デバッグだからわざと超大きい値にしてる、、、？ならなんでもないです。。。
レイアウトとか見るために、そこそこあり得る値が良いなあ、なんて。

Reply:
3704ef9